assets/filter-hub-tags.js

```diff
@@ -22,7 +22,9 @@ if (pagination == "true") {
   options.pagination = true;
 }
 
-var hubList = new List(listId, options);
+var newsList = new List(listId, options);
+
+var hubList = new List("hub-cards", options);
```

Comment:
I don't think you have to change `listId` to "hub-cards." The `listId` variable allows us to use the sort functionality on both the hub index and research model pages. By hard-coding the name of the variable here, I think you break the sorting functionality on the research model page. I'm not seeing all 26 cards appear now.

Also, it doesn't look like `newsList` is being used.